test/src/controllers/common/typeOfBusinessController.test.ts

```diff
@@ -35,6 +35,16 @@ describe("GET " + TYPE_OF_BUSINESS, () => {
         expect(res.text).toContain("What type of business are you registering?");
     });
 
+    it("should return status 200", async () => {
```

Comment:
Test coverage is 60%. Need to add more tests please.
Please check sonar report here.
[Sonar analysis report for acsp-web](http://code-analysis.platform.aws.chdev.org:9000/dashboard?id=uk.gov.companieshouse%3Aacsp-web&pullRequest=502)